### DISCUSSION TITLE
站点管理总是提示：新增站点失败：用户未通过认证，无法使用站点功能！

### DISCUSSION THREAD

**Discussion by szhongjie** [jxxghp/MoviePilot#2220] [问题咨询]
![11](https://github.com/jxxghp/MoviePilot/assets/50044522/4c36b794-9c60-45fe-a909-4dec163dbea4)
配置如图

**Comment by szhongjie** [comment]
这个设置有问题吗？添加站点的时候为什么还是提示未通过认证？

**Reply by cng10001** [reply]
套件能填环境变量了吗 认证应该是在环境变量里的

**Reply by szhongjie** [reply]
群晖安装之后有一个moviepilot配置图标，在里面直接修改的

**Comment by siruoren** [comment]
我也是同样问题，不知道怎么解决

**Reply by szhongjie** [reply]
我的解决啦

**Reply by comcj** [reply]
AUTH_SITE=iyuu
IYUU_SIGN=
这两个是什么，哪里找？？

**Reply by cng10001** [reply]
https://wiki.movie-pilot.org/configuration

**Reply by comcj** [reply]
iyuu	IYUU_SIGN：IYUU登录令牌  这个具体是什么，馒头站用什么？

**Reply by cng10001** [reply]
**AUTH_SITE认证站点支持：iyuu/hhclub/audiences/hddolby/zmpt/freefarm/hdfans/wintersakura/leaves/ptba /icc2022/xingtan/ptvicomo/agsvpt/hdkyl/qingwa/discfan/haidan/rousi**

只有认证站点支持的才能认证，馒头不在里面，不能认证

**Reply by szhongjie** [reply]
套间里安装iyuu按照说明做相应设置，就可以通过啦

**Reply by xulianglikejava** [reply]
使用的令牌是微信扫码后出现的流派吗 ？ 我iyuu绑定了站点还是不能用 绑定的也要是合作站点吗

**Reply by MikeYao0513** [reply]
> 我的解决啦

你解决了也不说怎么解决的，急死人啊！
我iyuu日志里显示绑定成功，但是无法添加站点，一直是用户未通过！
求方案，我是HDFANS，认证站点里有的

**Comment by mindcont** [comment]
馒头 能添加站点吗

**Reply by winston-cn** [reply]
说一下问题吧（版本1.9.17  win系统）
1. AUTH_SITE=iyuu可以写在mp的配置文件里，IYUU_SIGN必须是系统级的环境变量
2. iyuu申请token后，其实并未激活，要再找iyuu的合作站点激活（感觉在套娃！）
3. 认证不通过后，会一直启动新的服务来进行认证，导致win右下角全是MP的图标

**Reply by suerb** [reply]
> 说一下问题吧（版本1.9.17 win系统）
> 
> 1. AUTH_SITE=iyuu可以写在mp的配置文件里，IYUU_SIGN必须是系统级的环境变量
> 2. iyuu申请token后，其实并未激活，要再找iyuu的合作站点激活（感觉在套娃！）
> 3. 认证不通过后，会一直启动新的服务来进行认证，导致win右下角全是MP的图标

2 请问怎样找 iyuu 合作站点认证？

**Comment by godkingking** [comment]
一直提示 haidan认证出错：parameter passkey invalid，何解
![image](https://github.com/user-attachments/assets/62c8e77a-3ade-4e33-91bf-e7efa3ce2f7a)

**Comment by wangkt-tech** [comment]
馒头认证不了吗？

**Comment by godkingking** [comment]
可以了，看了攻略，密码输错了
&nbsp;


杰
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年11月17日(星期天) 晚上9:18
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [jxxghp/MoviePilot] 站点管理总是提示：新增站点失败：用户未通过认证，无法使用站点功能！ (Discussion #2220)














 
馒头认证不了吗？
